components/ComparisonsDisplay.vue

```diff
@@ -49,6 +49,10 @@ export default class ComparisonsDisplay extends Vue {
 
   cumulateTime: boolean = true;
 
+  subsplitLabel(name: string) {
+    return(name.startsWith('-') ? (name.substring(1) + " (subsplit)") : name);
```

Comment:
You don't need parenthesis when returning, return is not a function, also we're using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) pretty much everywhere so we can do that here too
```suggestion
    return name.startsWith('-') ? `${name.substring(1)} (subsplit)` : name;
```